geoportal/c2cgeoportal_geoportal/scripts/create_demo_theme.py

```diff
@@ -43,38 +45,41 @@ def main():
         description="Create and populate the database tables."
     )
     fill_arguments(parser)
-
     options = parser.parse_args()
 
-    get_app(options, parser)
-
-    from c2cgeoportal_commons.models import DBSession
-    from c2cgeoportal_commons.models.main import Interface, OGCServer, Theme, LayerGroup, LayerWMS
+    config_uri = get_config_uri(options)
+    loader = get_config_loader(config_uri)
+    loader.setup_logging()
+    settings = loader.get_wsgi_app_settings()
 
-    session = DBSession()
+    generate_mappers()
+    engine = get_engine(settings)
+    session_factory = get_session_factory(engine)
```

Comment:
All those are copied in "all" scripts, they should be in the `__init__` file.

In a function like `get_settings_[and_]session(...)`.